functions/dialogflow/functions/.eslintrc.json

```diff
@@ -0,0 +1,123 @@
+{
```

Comment:
You shouldn't need this file, we're using the top level config.

Reply:
Removed.